cmd/reset/reset.go

```diff
@@ -63,6 +63,14 @@ func (c *command) reset() error {
 		logrus.Fatal("k0s seems to be running! please stop k0s before reset.")
 	}
 
+	nodeCfg, err := c.K0sVars.NodeConfig()
+	if err != nil {
+		return err
+	}
+	if nodeCfg.Spec.Storage.Kine != nil && nodeCfg.Spec.Storage.Kine.DataSource != "" {
+		logrus.Warn("Kine dataSource is configured. k0s will not reset the data source. If you plan to continue using the data source, you should reset it to avoid conflicts.")
```

Comment:
hmm, IMO this is not 100% correct. In case kine uses local SQLite DB in `/var/lib/k0s/...` it WILL BE cleaned. Maybe something like `k0s will not reset the data source if it points to external database...`

Reply:
Done